tests/test_earthspy.py

```diff
@@ -330,6 +330,20 @@ def test_sentinelhub_request(self) -> None:
         # # check that a Sentinel Hub request was created
         assert isinstance(sr2, shb.SentinelHubRequest)
 
+    def test_geojson(self) -> None:
+        """Test geojson files"""
+
+        with open("earthspy/data/ilulissat.geojson", "r+") as files_geo:
+            data = json.load(files_geo)
+            array_coord = np.array(data["features"][0]["geometry"]
+                                       ['coordinates'][0])
```

Comment:
that's a good basis! Now we need to check this on all files that are in `earthspy/data`. At the moment I agree there's only one, but the point of the test here is to check that all files (including potential new ones) follow the right structure.